oak_functions/README.md

```diff
@@ -137,39 +137,6 @@ requirements, before publishing the results by logging to stdout.
 The Remote Attestation protocol implemented in Oak is currently integrated in
 Oak Functions.
 
-## Comparison with Oak
```

Comment:
I think it would be useful to add some details from the deleted documentation here. For instance how to run an application or create configuration files. There is also some technical background that could be relevant, for instance about the use of WebAssembly. I'd suggest creating an issue to do this in a separate PR.

Reply:
Thanks, created https://github.com/project-oak/oak/issues/2541